react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
+
+  let validPassLength = password.length >= 8;
+  let validPass = validPassLength ? 'is-valid' : 'is-invalid';
+  let errText =
+    password.length === 0
+      ? 'A password is required.'
+      : 'Your password is too short.';
+
+  return (
+    <form>
```

Comment:
You are building a reusable component that you might want to put into a form. So you don't want the `<form>` element here. It will be in the parent.